main.go

```diff
@@ -3,66 +3,78 @@ package main
 import (
 	"encoding/json"
 	"net/http"
+	"os"
 	"sort"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/asdine/storm"
 	"github.com/jmoiron/jsonq"
 	"github.com/resin-io/edge-node-manager/api"
 	"github.com/resin-io/edge-node-manager/application"
 	"github.com/resin-io/edge-node-manager/config"
+	"github.com/resin-io/edge-node-manager/device"
 	"github.com/resin-io/edge-node-manager/process"
+	"github.com/resin-io/edge-node-manager/radio/bluetooth"
 	"github.com/resin-io/edge-node-manager/supervisor"
 )
 
-// This variable will be populated at build time with the current version tag
-var version string
+var (
+	version   string // This variable will be populated at build time with the current version tag
+	loopDelay time.Duration
```

Comment:
I would also add a little description of what this variable is for, as it's not a local.